deploy/helm/sumologic/conf/metrics/otelcol/config.yaml

```diff
@@ -333,14 +337,18 @@ service:
         - resource/delete_source_metadata
         - sumologic_schema
         - batch
+{{- if eq .Values.sumologic.metrics.sourceType "http" }}
         - routing
+{{- end }}
       exporters:
         - sumologic/default
+{{- if eq .Values.sumologic.metrics.sourceType "http" }}
         - sumologic/apiserver
         - sumologic/control_plane
         - sumologic/controller
         - sumologic/kubelet
         - sumologic/node
         - sumologic/scheduler
         - sumologic/state
+{{- end }}
```

Comment:
Will that work fine (without any blacklisting and throttling due to metrics cardinality)?

Reply:
In the meantime, can you review these changes on the assumption that this ok to do?

Reply:
Yes, remaining changes looks fine for me